.github/workflows/dapr_cli.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
     name: Build ${{ matrix.target_os }}_${{ matrix.target_arch }} binaries
     runs-on: ${{ matrix.os }}
     env:
-      GOVER: 1.19.2
+      GOVER: "^1.19.3"
```

Comment:
Why this change, shouldn't we pin to a known version? This will take up all minor and patch upgrades automatically. I propose we stick to pinning a version to avoid any unforeseen problems.

Reply:
see https://github.com/npm/node-semver#caret-ranges-123-025-004

Reply:
Yes, so this means >=1.19.3 and <2.0.0-0, right?